spynnaker8/models/connectors/from_list_connector.py

```diff
@@ -34,11 +38,15 @@ def __init__(
             if given, a callable that display a progress bar on the terminal.
         """
         # pylint: disable=too-many-arguments
-        if conn_list is None or not len(conn_list):
+        if conn_list is None:
             raise InvalidParameterType(
                 "The connection list for the FromListConnector must contain"
-                " at least a list of tuples, each of which should contain at "
-                "least: (pre_idx, post_idx)")
+                " either a list of tuples, each of which should contain at "
+                "least: (pre_idx, post_idx), or be an empty list")
+
+        if not len(conn_list):
```

Comment:
Can `conn_list` be a numpy array? If not, use `if not conn_list:` (Python general style)